Detectors/MUON/MCH/Raw/ElecMap/src/elecmap.py

```diff
@@ -87,7 +87,7 @@ def gencode_do(df, df_cru, solar_map, chamber):
     for row in df_cru.itertuples():
         if len(row.solar_id) > 0:
             out.write("add_cru(s2f,{},{},{});\n".format(
-                row.fee_id, row.link_id, row.solar_id))
+                row.fee_id, int(row.link_id)%12, row.solar_id))
```

Comment:
This is not strictly needed with the latest version of the mapping on the web, but we can keep it as an extra safety.